integration_test/third_party_apps_test/main_test.go

```diff
@@ -624,26 +624,41 @@ func runSingleTest(ctx context.Context, logger *logging.DirectoryLogger, vm *gce
 		if app == "mariadb" {
 			instrumentedApp = "mysql"
 		}
-		for _, m := range metadata.ExpectedMetrics {
+		for _, m := range integrationMetadata.ExpectedMetrics {
 			// The windows metrics that do not target workload.googleapis.com cannot set
 			// the instrumentation_* labels
 			if strings.HasPrefix(m.Type, "agent.googleapis.com") {
 				continue
 			}
 			if m.Labels == nil {
-				m.Labels = map[string]string{}
+				m.Labels = []*metadata.MetricLabel{}
```

Comment:
Wouldn't this code be much shorter if we kept `m.Labels` as a map?

Reply:
It would, but then we wouldn't be able to reuse labels via anchors/aliases.

Reply:
Do you mean YAML anchors/aliases? I didn't see any such reuse in this PR, so it feels like added complexity without realizing its benefits.

Reply:
You missed them in `dcgm/metadata.yaml` (e.g., `model_label`).

Reply:
Whoops, you're right -- my mistake!